src/smart/SmartPatient.js

```diff
@@ -94,6 +94,7 @@ export function SmartPatient() {
   } else {
     return (
       <div className="content">
+      <p className="alert alert-danger">The CDC/MITRE Cervical Cancer CDS Dashboard is under pilot evaluation and is <b>not for use in clinical practice.</b></p>
```

Comment:
Sent comment about wording over Slack.

Reply:
Resolved over Slack. We can update the message in the future in response to clinician pilot feedback, if needed.